MaxText/pyconfig.py

```diff
@@ -51,6 +51,19 @@ def validate_attention_type(s: str) -> None:
       "Invalid attention type was passed. Valid options ", valid_attention_types
     )
 
+def maybe_warn_skip_remat_names(keys):
+  if keys['attention'] in ('flash', 'cudnn_flash_te'):
+    skip_remat_names = ()
+    if keys['remat_policy'] in ('proj', 'full'):
```

Comment:
I think FlashAttention is so dominant as a solution that I'm not interested in these inferior checkpointing strategies because they only serve to confuse. Can you provide justification for why they are necessary?

Reply:
You are absolutely right. Removed all the remat within `AttentionOp` (where we use FlashAttention instead). By the way, mlperf runs align your insights here, we used FlashAttention and quantize the other matmuls.